### DISCUSSION TITLE
Getting error while make x11iraf in ubuntu 20.04

### DISCUSSION THREAD

**Discussion by BOOM5s** [iraf-community/iraf#310] [Q&A]
`Warning, mkpkg line 13: module `libpkg.a' not found or returned error
xc -c -x x_ism.x
envinit: cannot open `iraf$extern/.zzsetenv.def'
envinit: cannot open `iraf$extern/.zzsetenv.def'
x_ism.x:13:10: fatal error: f2c.h: No such file or directory
compilation terminated.
xc -Nz   -z x_ism.o libpkg.a -o ism_wcspix.e -lds -lxtools -liminterp -lslalib
envinit: cannot open `iraf$extern/.zzsetenv.def'
Warning: file `x_ism.o' not found
Warning: file `libpkg.a' not found
gcc: error: libmain.o: No such file or directory
gcc: error: libex.a: No such file or directory
gcc: error: liblapack.a: No such file or directory
gcc: error: libsys.a: No such file or directory
gcc: error: libvops.a: No such file or directory
gcc: error: libVOTable.a: No such file or directory
Warning, mkpkg line 3: module `relink' not found or returned error `

**Comment by olebole** [comment]
The error message means that the build didn't find your IRAF installation (f.e. if the `iraf` environment variable was not set).
As long as you don't neet ximtool or can live without WCS support in ximtool, you can ignore this. xgterm is already built and can be used (just copy it to your PATH).
For Ubuntu 20.04, there are also pre-built packages: [xgterm](https://launchpad.net/~olebole/+archive/ubuntu/iraf/+files/xgterm_2.0+2020.06.15+dfsg-1~focal+1_amd64.deb) and [ximtool](https://launchpad.net/~olebole/+archive/ubuntu/iraf/+files/ximtool_2.0+2020.06.15+dfsg-1~focal+1_amd.deb). Using these makes them updateable when you migrate to a more modern version.
